### DISCUSSION TITLE
GitHub test error - STATA code, unable to open log file

### DISCUSSION THREAD

**Discussion by AnnaB45** [opensafely/documentation#172] [Q&A] [answered]
The Immunosuppressant-meds-research master branch is failing at the "Run model" stage ([details here](https://github.com/opensafely/immunosuppressant-meds-research/runs/2032474330)).
The analysis is in STATA and I think these are the error details:
`. log using "$logdir/cr_create_analysis_dataset", replace`
`(note: file /workspace/logs/cr_create_analysis_dataset.smcl not found)`
`file /workspace/logs/cr_create_analysis_dataset.smcl could not be opened`
`r(603);`

The STATA code is able to open a log file when I run this locally, any ideas as to why it might be failing on GitHub?

**Comment by alexwalkerepi** [comment, marked as answer]
The current failure is I think because there's no `logs/` directory in the root of the repo, which is where it's trying to write it. Try adding the folder along with a `.gitkeep` file, in the same way as the `output/` folder here https://github.com/opensafely/immunosuppressant-meds-research/tree/master/output.

**Reply by AnnaB45** [reply]
Thanks Alex
